drivers/espi/espi_mchp_xec.c

```diff
@@ -417,24 +417,25 @@ static int espi_xec_send_oob(struct device *dev, struct espi_oob_packet pckt)
 	LOG_DBG("%s\n", __func__);
 
 	if (!(ESPI_OOB_REGS->TX_STS & MCHP_ESPI_OOB_TX_STS_CHEN)) {
-		LOG_WRN("OOB channel is disabled\n");
+		LOG_ERR("OOB channel is disabled");
 		return -EIO;
 	}
 
 	if (ESPI_OOB_REGS->TX_STS & MCHP_ESPI_OOB_TX_STS_BUSY) {
-		LOG_WRN("OOB channel is busy\n");
+		LOG_ERR("OOB channel is busy");
 		return -EBUSY;
 	}
 
 	if (pckt.len > MAX_OOB_BUFFER_SIZE) {
+		LOG_ERR("insufficient space");
 		return -EINVAL;
 	}
 
 	memcpy(slave_tx_mem, pckt.buf, pckt.len);
 
 	ESPI_OOB_REGS->TX_LEN = pckt.len;
 	ESPI_OOB_REGS->TX_CTRL = MCHP_ESPI_OOB_TX_CTRL_START;
-	LOG_DBG("%s %d\n", __func__, ESPI_OOB_REGS->TX_LEN);
+	LOG_ERR("%s %d", __func__, ESPI_OOB_REGS->TX_LEN);
```

Comment:
I think this should be `LOG_DBG()` instead.

Reply:
fixed